### PR TITLE
More CTAs for my articles

### PR DESCRIPTION
**Pull request by adamgordonbell**

### CODE REVIEW

**Review by dirien — APPROVED** [review]
LGTM!

File: content/blog/cursed-container-iceberg/index.md [hunk 1]
```diff
@@ -7,11 +7,13 @@ meta_image: meta.png
 authors:
     - adam-gordon-bell
 tags:
-    - k8s, containers, cloud-native, docker
+    - k8s
```
Comment (thoward) [inline content/blog/cursed-container-iceberg/index.md:10]: we should probably also include a tag that is `kubernetes` spelled out

File: content/blog/cursed-container-iceberg/index.md [hunk 2]
```diff
@@ -90,6 +92,16 @@ Let's plunge deeper. Because one Kubernetes cluster is never enough, why not spr
 - **Nutanix Karbon** - Because regular Kubernetes wasn't converged enough
 - **Metal Stack Cloud Kubernetes** - For when virtual machines are too virtual
 
+{{% notes type="tip" %}}
+
+**You Might Also Like:**
```
Comment (thoward) [inline content/blog/cursed-container-iceberg/index.md:97]: We use sentence case for headings.

File: content/blog/cursed-container-iceberg/index.md [hunk 3]
```diff
@@ -148,6 +160,22 @@ There you have it, the container iceberg. And I didn't even have room to add the
 
 - **GitHub Actions et al.** – Ever wanted to run a production service disguised as a CI job? Just label your production API start-up script as "integration_test_step_42", add a `sleep 21600` and a call to retrigger itself, and hope nobody notices your "test" has been running for months. Apply this same trick to AWS CodeBuild, GitLab CI/CD, CircleCI, Azure Pipelines, Travis CI, Bitbucket Pipelines, Drone CI, Jenkins Cloud, TeamCity Cloud, and Google Cloud Build for Bonus points.
 
+## Managing Your Container Zoo with Pulumi
```
Comment (thoward) [inline content/blog/cursed-container-iceberg/index.md:163]: Use sentence casing for all headings (I won't comment on each one).

**Review by thoward — APPROVED** [review]

File: content/blog/cursed-container-iceberg/index.md (hunk 2 above)
Comment (adamgordonbell) [reply content/blog/cursed-container-iceberg/index.md:97]: Oh no, I've got a lot of these  ...

**Pull request merged**